apis/python/tests/test_registration_mappings.py

```diff
@@ -399,17 +407,29 @@ def test_isolated_soma_experiment_mappings(soma1):
     ).data == (6, 3, 4)
 
 
+@pytest.mark.parametrize("obs_field_name", ["obs_id"])
+@pytest.mark.parametrize("var_field_name", ["var_id"])
 @pytest.mark.parametrize("permutation", [[0, 1, 2, 3], [2, 3, 0, 1], [3, 2, 1, 0]])
 @pytest.mark.parametrize("solo_experiment_first", [True, False])
 def test_multiples_without_experiment(
     tmp_path,
-    h5ad1,
-    h5ad2,
-    h5ad3,
-    h5ad4,
+    obs_field_name,
+    var_field_name,
     permutation,
     solo_experiment_first,
 ):
+    h5ad1 = create_h5ad_canned(1, obs_field_name, var_field_name)
+    h5ad2 = create_h5ad_canned(2, obs_field_name, var_field_name)
+    h5ad3 = create_h5ad_canned(3, obs_field_name, var_field_name)
+    h5ad4 = create_h5ad_canned(4, obs_field_name, var_field_name)
+
+    import shutil
+
+    shutil.copy(h5ad1, "/tmp/1.h5ad")
```

Comment:
should this be a `TemporaryDirectory`, instead of hard-coded `/tmp`?

Reply:
Derp, those were debugs to take back out. Thanks!